css/impact.css

```diff
@@ -245,42 +244,48 @@ Feel free to change the min/max-width values to whatever you need. Use variables
 		margin: 0 auto;
 	}
 	/* tech around section */
-	.red-block-one {
+	/* red block */
+	.tech-img.style-left::before {
+		transform: translate(-15%, -15%);
+	}
+	.tech-img::before {
 		background-color: var(--color-amaranth);
 		border-radius: var(--spacing10);
-		display: block;
-		min-width: 200px;
-		min-height: 200px;
 		position: absolute;
-		transform: translate(-45px, -235px);
+		content: " ";
+		width: 80%;
+		aspect-ratio: 1/1;
```

Comment:
nice use of aspect ratio to get rid of some hard coded values!

Reply:
thank you kindly!